src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -356,6 +404,9 @@ void InitSurfaceHeatBalance(EnergyPlusData &state)
         }
     }
 
+    // yujie: variable thermal solar absorptance overrides
+    MaterialAddOnOverride(state);
```

Comment:
Not a descriptive name for a the function. Maybe `UpdateVariableAbsorptances` would be better. Currently, the function name is not telling you what about the material is being overridden.

Reply:
This is indeed a more descriptive name